grails-app/assets/javascripts/asinha/customer-create-httpPOST.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function () {
+    $("form").on("submit", function(event) {
+        event.preventDefault();
+        let infosCustomer = {};
+        let data = new FormData(document.querySelector("form"));
+        data.forEach(function (value,key) {
+            infosCustomer[key] = value;
+        });
+        delete infosCustomer.cpfCnpjRadio;
```

Comment:
se fizer assim não precisa fazer o `delete` depois
```suggestion
        data.forEach(function (value,key) {
            if (key != "cpfCnpjRadio") infosCustomer[key] = value;
        });
```